src/test/resources/ci/scripts/run.sh

```diff
@@ -155,7 +155,7 @@ main() {
 
     # Start the IDE.
     echo -e "\n$(${currentTime[@]}): INFO: Starting the IntelliJ IDE..."
-    ./gradlew -Dide.slow.operations.assertion=false runIdeForUiTests --info  > remoteServer.log  2>&1 &
```

Comment:
Paul actually made a change to this in PR https://github.com/OpenLiberty/liberty-tools-intellij/pull/673/files as part of issue 590. Therefore, at the time, he might faced the slow operation exception, perhaps that was the reason he added this change in order to avoid getting exceptions during UI testing.

So, I made a change to the initial state with the default value:

`./gradlew runIdeForUiTests --info > remoteServer.log 2>&1 &`
@turkeylurkey , could you please confirm if this change is okay with the modifications I made?